chapters/en/module3.md

```diff
@@ -12,19 +12,708 @@ id: 3
+Comedy</text></g></g><path class="foreground" aria-hidden="true" d="" pointer-events="none" display="none"/></g><g transform="translate(0,130)"><path class="background" aria-hidden="true" d="M0,0h168v11h-168Z" pointer-events="none" opacity="1"/><g><g class="mark-symbol role-legend-symbol" pointer-events="none"><path transform="translate(6,6)" d="M5,0A5,5,0,1,1,-5,0A5,5,0,1,1,5,0" fill="#4c78a8" stroke-width="1.5" opacity="1"/></g><g class="mark-text role-legend-label" pointer-events="none"><text text-anchor="start" transform="translate(16,9)" font-family="sans-serif" font-size="10px" fill="#000" opacity="1">Thriller/Suspense</text></g></g><path class="foreground" aria-hidden="true" d="" pointer-events="none" display="none"/></g><g transform="translate(0,143)"><path class="background" aria-hidden="true" d="M0,0h168v11h-168Z" pointer-events="none" opacity="1"/><g><g class="mark-symbol role-legend-symbol" pointer-events="none"><path transform="translate(6,6)" d="M5,0A5,5,0,1,1,-5,0A5,5,0,1,1,5,0" fill="#f58518" stroke-width="1.5" opacity="1"/></g><g class="mark-text role-legend-label" pointer-events="none"><text text-anchor="start" transform="translate(16,9)" font-family="sans-serif" font-size="10px" fill="#000" opacity="1">Western</text></g></g><path class="foreground" aria-hidden="true" d="" pointer-events="none" display="none"/></g></g></g><path class="foreground" aria-hidden="true" d="" pointer-events="none" display="none"/></g></g><g class="mark-text role-legend-title" pointer-events="none"><text text-anchor="start" transform="translate(0,9)" font-family="sans-serif" font-size="11px" font-weight="bold" fill="#000" opacity="1">Major
+Genre</text></g></g><path class="foreground" aria-hidden="true" d="" pointer-events="none" display="none"/></g></g></g><path class="foreground" aria-hidden="true" d="" display="none"/></g></g></g></svg>
+
+<choice id="1" >
+<opt text="There are too many records.">
+
+Having a lot of observations has no impact on a density plot. 
+
+</opt>
+
+<opt text="The majority of the observations lie to the left of the axis making the values and genres difficult to observe." correct="true">
+
+Got it! 
+
+</opt>
+
+<opt text="Their are too many genres to compare." >
```

Comment:
I think too many genres (categories) is problematic for densities on a single plot...

Reply:
Oops! I've change this to: 
> "Their are too many features in the dataframe.

> INCORRECT: We are only plotting one feature from a dataframe so this should not matter.